Trend Micro/trend-micro-vision-one/ingest/parser.yml

```diff
@@ -0,0 +1,85 @@
+name: trend-micro-vision-one
+ignored_values: []
+pipeline:
+  - name: parsed_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: alert
+          event.category: ["intrusion_detection"]
+          event.type: ["info"]
+          observer.vendor: "TrendMicro"
+          observer.product: "Vision One"
+
+          event.reason: "{{parsed_event.message.model}}"
+
+      - set:
+          "@timestamp": "{{parsed_event.message.createdDateTime}}"
+
+          organization.name: "{{ (parsed_event.message.impactScope.entities | selectattr('entityType', 'equalto', 'account') | first).entityValue }}"
+          organization.id: "{{ (parsed_event.message.impactScope.entities | selectattr('entityType', 'equalto', 'account') | first).entityId }}"
```

Comment:
According to the tests, I wonder if this information should not go into `user.domain`,  `user.name` and `user.id`?